scripts/us_census/pep/Population_Estimate_by_ASR/common_functions.py

```diff
@@ -0,0 +1,75 @@
+"""
+Contains the recurring functions of other files
+"""
+import json
+import os
+import pandas as pd
+
+
+def _input_url(file_name: str, key_name: str):
```

Comment:
for functions imported and used in other files:
- please remove the `_` prefix
- add types to the function arguments and return type
  `def input_url(file_name: str, key_name: str): -> list:`
- indicate arguments, returned value sand exceptions raised if any, in the function block comment

Reply:
Removed _ to move from protected to public